domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt

```diff
@@ -750,13 +776,344 @@ class TopicControllerTest {
     assertThat(questionListResultCaptor.value.isFailure()).isTrue()
   }
 
+  @Test
+  @ExperimentalCoroutinesApi
+  fun testGetTopic_invalidTopicId_getTopic_noResultFound() =
+    runBlockingTest(coroutineContext) {
+      topicController.getTopic(profileId1, INVALID_TOPIC_ID_1).observeForever(mockTopicObserver)
+      advanceUntilIdle()
+
+      verifyTopicFailed()
```

Comment:
Is this actually verifying that ``getTopic`` failed? I suggest clarifying in the name of all your helpers which operation it corresponds to for both failure & success cases.

Reply:
Updated to 
`verifyGetTopicSucceeded`, `verifyGetStorySucceeded` and similarly at all other places